src/rubrix/__init__.py

```diff
@@ -115,6 +115,20 @@ def log(
         records=records, name=name, tags=tags, metadata=metadata, chunk_size=chunk_size
     )
 
+def copy(dataset:str, dest:str):
```

Comment:
I think I prefer something more explicit for the `dest` parameter. Maybe `name_of_copy`, I think most of the time you will use this method without kwargs, that is simply `rb.copy("dataset", "dataset_copy")`. I would also provide a default value `None`, in which case the name of the copy would be something like `{source_name}_copy`.

Reply:
done

Reply:
> I would also provide a default value None, in which case the name of the copy would be something like {source_name}_copy

I prefer to keep as is. Making this default behaviour and so own could bring extra logic. What about if already exists a `{source_name}_copy` dataset? client could be asign a new `{source_name}_copy2` name, and so on.

Once this method is validated and we're sure it will be part of client API for a while, we can improve with proposal like yours.